torch/_torch_docs.py

```diff
@@ -3561,6 +3561,38 @@ def merge_dicts(*dicts):
     RuntimeError: bool value of Tensor with no values is ambiguous
 """.format(**common_args))
 
+add_docstr(torch.kron,
+           r"""
+kron(input, other, *, out=None) -> Tensor
+
+Computes the Kronecker product of :attr:`input` and :attr:`other`.
+
+If :attr:`input` is a :math:`(n \times m)` tensor, :attr:`other` is a
+:math:`(k \times l)` tensor, the result will be a :math:`(n*k \times m*l)` tensor.
+
+Supports real and complex inputs.
+
+Arguments:
+    input (Tensor): the first tensor to be multiplied
+    other (Tensor): the second tensor to be multiplied
+
+Keyword args:
+    out (Tensor, optional): The output tensor. Ignored if ``None``. Default: ``None``
+
+Returns:
```

Comment:
Kill this "Returns:" bit. We typically only use it when the returned value is more complicated, like a tuple of multiple tensors.